devices/src/legacy/rtc_pl031.rs

```diff
@@ -203,18 +205,17 @@ mod tests {
         assert_eq!(v, 123);
 
         // Read and write to the LR register.
-        let v = time::get_time();
-        LittleEndian::write_u32(&mut data, v.sec as u32);
+        let v = fc_util::get_time(fc_util::ClockType::Real);
```

Comment:
Instead of writing fc_util::something, please import what you need from fc_util.